web_service.go

```diff
@@ -8,6 +8,8 @@ import (
 	"github.com/emicklei/go-restful/log"
 )
 
+const DefaultRootPath = "/"
```

Comment:
I did agree that we shouldn't expose a const here, so just remove it. This PR will not change the default behavior of `WebService`, but just fix an edge case when you pass an empty string to `Path`.